### PR TITLE
Add back background service health check

### PR DESCRIPTION
**Pull request by abdullah248**
## Description
Adds back health check for background service but move application insights telemetry adding from core to api.

Also fixed a typo in a doc (apparently png and PNG are treated the same in visual studio code but not on github)

### CODE REVIEW

File: docs/development/setup.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Postman only supports uploading DICOM files using the single part payload define
    - Content-Type: `application/dicom`
 - Click `Send`, a successful post will result in a 200 OK
 
-![Postman headers](/docs/images/postman-singlepart-headers.png)
+![Postman headers](/docs/images/postman-singlepart-headers.PNG)
```
Comment (wsugarman) [inline docs/development/setup.md:56]: Was this intentional?

File: src/Microsoft.Health.Dicom.Api/Microsoft.Health.Dicom.Api.csproj [hunk 2]
```diff
@@ -14,6 +14,7 @@
     <PackageReference Include="fo-dicom" Version="4.0.7" />
     <PackageReference Include="fo-dicom.Json" Version="4.0.7" NoWarn="NU1701" />
     <PackageReference Include="MediatR" Version="9.0.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.17.0" />
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.Api/Microsoft.Health.Dicom.Api.csproj:17]: I was wrong! Looks like we need some additional package specifications for Component Governance for Microsoft.AspNetCore.Http and System.Text.Encodings.Web:

```xml
<PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.22" PrivateAssets="All" />
<PackageReference Include="System.Text.Encodings.Web" Version="4.5.1" PrivateAssets="All" />
```

You can double-check this by checking the build's CG alerts inside of the build logs.

File: docs/development/setup.md (hunk 1 above)
Comment (abdullah248) [reply docs/development/setup.md:56]: haha yea, appartenlty visual studio code treats png and PNG the same but github does not. Its fixing a previous pr

File: src/Microsoft.Health.Dicom.Api/Microsoft.Health.Dicom.Api.csproj (hunk 2 above)
Comment (abdullah248) [reply src/Microsoft.Health.Dicom.Api/Microsoft.Health.Dicom.Api.csproj:17]: Added them in

**Review by wsugarman — APPROVED** [review]

**Pull request merged**